e2e/tests/components/number-picker/number-picker.e2e-spec.ts

```diff
@@ -156,4 +156,22 @@ describe('Number Picker Tests', () => {
         expect(await page.confirmUpDownControlIsDisabled(page.numberPicker2, 'down')).toBeFalsy();
 
     });
+
+    it('should not continue to display ux-number-picker-invalid class when value changed from above max value to max value', async () => {

```

Comment:
Could you combine these into a "changes to min and max should update the validation state" test case? End to end tests in this project are meant to be larger user scenarios, preferably no more than one per bug fix.